sycl/test-e2e/SYCLBIN/link_mixed_opt_input.cpp

```diff
@@ -17,7 +17,7 @@
 // once fixed.
 // REQUIRES: target-spir
 
-// RUN: %clangxx --offload-new-driver -fsyclbin=input -fsycl-allow-device-image-dependencies -O0 %S/Inputs/exporting_function.cpp -o %t.export.syclbin
+// RUN: %clangxx --offload-new-driver -fsyclbin=input -fsycl-allow-device-image-dependencies %if cl_options %{/Od%} %else %{-O0%} %S/Inputs/exporting_function.cpp -o %t.export.syclbin
```

Comment:
FYI: We have `%O0` which does the trick:

https://github.com/intel/llvm/blob/ea655093b293c60b1c3789156e253e914173e7d7/sycl/test-e2e/lit.cfg.py#L579

https://github.com/intel/llvm/blob/ea655093b293c60b1c3789156e253e914173e7d7/sycl/test-e2e/lit.cfg.py#L605